test/tests_helix.zig

```diff
@@ -156,3 +157,66 @@ test "long_words_movement" {
         try apply_movements(move.moves, root, &the_cursor, metrics(), move.row, move.col);
     }
 }
+test "to_char_right_beyond_eol" {
+    const buffer = try Buffer.create(a);
+    defer buffer.deinit();
+    buffer.update(try buffer.load_from_string(doc, &eol_mode, &sanitized));
+    const root: Buffer.Root = buffer.root;
+    the_cursor.col = 0;
+    the_cursor.row = 0;
+    const expected_error = error.Stop;
+    const ctx_a = fmt_command(.{"a"});
```

Comment:
@neurocyte I think I'm not properly filling the context, my intention is to encode 'a' to be found later when the decoding is done. Test is failing in line 181 because when the argument of the command is decoded does not match with the one I hadn't set correctly in line 168.